lib/osf-components/addon/components/registries/registration-form-navigation-dropdown/component.ts

```diff
@@ -24,4 +32,11 @@ export default class RegistrationFormNavigationDropdown extends Component {
                 blockType === 'question-label'
             ) && displayText);
     }
+
+    didReceiveAttrs() {
+        assert('Registries::RegistrationFormNavigationDropdown requires @schemablocks!', Boolean(this.schemaBlocks));
+        this.metadataFields = Object.values(DraftMetadataProperties)
+            .filter(prop => prop !== DraftMetadataProperties.NodeLicenseProperty)
+            .map(underscore);
```

Comment:
this doesn't depend on any attrs -- how about put it in `constructor` instead?

also, then `metadataFields` never changes value and doesn't need `@tracked`, i think

Reply:
IIRC, Ember will throw an error if it's not `@tracked`.

Reply:
This should be ok in the constructor, since it doesn't depend on any args.

Reply:
Now that I think about it, this can just be the initializer:
```ts
metadataFields: string[] = Object.values(DraftMetadataProperties)
    .filter(prop => prop !== DraftMetadataProperties.NodeLicenseProperty)
    .map(underscore);
```

Reply:
hm, it'll throw an error for a non-tracked field assigned a value in the constructor?

Reply:
I'm not sure, actually. Would make sense for it not to...